addons/event_crm/views/event_crm_views.xml

```diff
@@ -0,0 +1,89 @@
+            </tree>
+        </field>
+    </record>
+
+    <record id="event_crm_rule_view_form" model="ir.ui.view">
+        <field name="name">event.crm.rule.view.form</field>
+        <field name="model">event.crm.rule</field>
+        <field name="arch" type="xml">
+            <form string="Lead Generation Rule">
+                <sheet>
+                    <widget name="web_ribbon" title="Archived" bg_color="bg-danger" attrs="{'invisible': [('active', '=', True)]}"/>
+                    <div class="oe_title">
+                        <label for="name" class="oe_edit_only"/>
+                        <h1><field name="name" placeholder="e.g. B2B Fairs"/></h1>
+                    </div>
+                    <field name="active" invisible="1"/>
+                    <group name="lead_creation" invisible="1">
+                        <field name="lead_creation_basis" widget="radio"/>
+                    </group>
+                    <group string="For any of these Events">
+                        <group>
+                            <field name="event_type_ids" widget="many2many_tags"/>
+                            <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
+                        </group>
+                        <group>
+                            <field name="event_id" options="{'no_create': True}"/>
+                        </group>
+                    </group>
+                    <group string="If the Attendees meet these Conditions">
+                        <field name="registration_filter_domain" widget="domain" options="{'model': 'event.registration'}" nolabel="1"/>
+                    </group>
+                    <group string="Lead Default Values">
+                        <group class="col">
+                            <field name="lead_type" groups="crm.group_use_lead"/>
+                            <field name="lead_sales_team_id"/>
+                            <field name="lead_user_id"/>
+                        </group>
+                        <group class="col">
+                            <field name="lead_tag_ids" widget="many2many_tags" options="{'color_field': 'color'}"/>
+                        </group>                         
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="action_event_crm" model="ir.actions.act_window">
+        <field name="name">Lead Generation Rule</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">event.crm.rule</field>
+        <field name="view_mode">tree,form</field>
+        <field name="help" type="html">
+            <p class="o_view_nocontent_smiling_face">Create a Lead Generation Rule</p>
+            <p>Those automatically create leads when attendees register.</p>
+        </field>
+    </record>
+
+    <menuitem name="Lead Generation" id="menu_event_crm" 
```

Comment:
And here event_crm_menu (you got the idea: the model is always first, then type of records, then additional information).